Sources/TensorFlow/Operators/NN.swift

```diff
@@ -343,18 +343,14 @@ func _vjpDepthwiseConv2D<Scalar: TensorFlowFloatingPoint>(
     filter: Tensor<Scalar>,
     strides: (Int, Int, Int, Int),
     padding: Padding
-) -> (Tensor<Scalar>, (Tensor<Scalar>) -> (Tensor<Scalar>, Tensor<Scalar>)) {
+    ) -> (Tensor<Scalar>, (Tensor<Scalar>) -> (Tensor<Scalar>, Tensor<Scalar>)) {
```

Comment:
This is a swift-format change @rxwei, Should i keep this or revert it? Because if it's to be kept, we should probably standardize this for the other functions as well.

Reply:
This does not seem like what swift-format would do. How did you get the swift-format? Could you build and run this [swift-format](https://github.com/apple/swift-format)?

Reply:
so when I run the command it's usually in the `swift-format.exe <file>` way.
I'll run the other swift-format if it's buildable on WSL.